portal-impl/src/com/liferay/portal/webserver/WebServerServlet.java

```diff
@@ -1476,6 +1473,15 @@ private static Group _getGroup(long companyId, String name)
 		return user.getGroup();
 	}
 
+	private static String _getPath(HttpServletRequest httpServletRequest) {
+		String path = httpServletRequest.getRequestURI();
+		String location =
+			httpServletRequest.getContextPath() +
+				httpServletRequest.getServletPath();
+
+		return path.substring(location.length() + 1);
```

Comment:
Shouldn't `path` be named `requestURI` and `location` be renamed to `path`?

Reply:
Completely agree. I just updated it. Thanks!.